planetcantile/data/generate.py

```diff
@@ -48,30 +48,46 @@ def __post_init__(self):
 with urllib.request.urlopen('https://raw.githubusercontent.com/pdssp/planet_crs_registry/main/data/result.wkts') as response:
     resp = response.read().decode(response.headers.get_content_charset())
 
-# Parse all the GeoCRSs from the list
-geocrss = []
+# Parse all the CRSs from the list
+allcrss = []
+acceptable_projections = ('Equirectangular', 'North Polar', 'South Polar')
 for wkt_str in resp.split(os.linesep + os.linesep):
-    if 'GEOGCRS' in wkt_str[:7] and 'TRIAXIAL' not in wkt_str:  # insanely hacky
-        geocrss.append(wkt_str)
+    if 'TRIAXIAL' not in wkt_str:  # insanely hacky
+        if wkt_str.startswith('GEOGCRS') or any(_ in wkt_str for _ in acceptable_projections):
+            allcrss.append(wkt_str)
 
 # Build a dynamic crss list
 crss = []
-for crs in geocrss:
+for crs in allcrss:
     crs_obj = CRS(crs)
     title = crs_obj.name
     auth = crs_obj.to_authority(min_confidence=25)
     if auth is not None:
         authority_version, code = auth
         authority, version = authority_version.split('_')
         identifier = f'{authority}_{code}_{version}'
+        geographic_crs = crs_obj.geodetic_crs
+        # get the extent, if clon == 180 we have a 0-360 longitude crs
+        extent = (0.0, -90.0, 360.0, 90.0) if 'clon = 180' in crs else (-180.0, -90.0, 180.0, 90.0)
+        # do some work to get extent from projected CRSs
+        if crs_obj.is_projected:
+            # todo: this doesn't work well with ographic geographic crss (those with inverse flattenings), fails ecq: Invalid latitude
+            # looks to be an issue where the x and y is swapped, if the inverse_flattening is not 0.0 we would have to swap
+            # this x and y swap is happening despite setting always_xy to True
+            # another way is to check the coordinate_system, I see westing instead of easting
+            if geographic_crs.coordinate_system.axis_list[0].abbrev == 'W':
+                extent = (extent[1], extent[0], extent[3], extent[2])
+            transformer = Transformer.from_crs(geographic_crs, crs_obj, authority='IAU', always_xy=True, allow_ballpark=False, accuracy=0.001)
+            # todo this still fails errcheck=True
+            extent = transformer.transform_bounds(*extent, densify_pts=51)
```

Comment:
Where is the 51 coming from?

Reply:
it's arbitrary, default is 21 points so figured adding some more couldn't hurt